moby/moby
# Adding the ability to configure default capabilities

Signed-off-by: zach <zachary@rocana.com>



**- What I did**
I added the ability to set a default set of linux capabilities to give to newly created containers. 

**- How I did it**
I extended the work  https://github.com/moby/moby/commit/80d7bfd54d19cc25f4855b4f2e9e4ad2e47f86c0 to add the ability to configure the docker daemon to set a more limited default set of capabilities.  
**- How to verify it**
I added a new check in the tests, but if you configure the caps via dockerd cli or json and run:
`capsh --print`
You will see the caps for your container.

**- Description for the changelog**

Adding the ability to configure default capabilities.

**- A picture of a cute animal (not mandatory but encouraged)**

![alt text](https://i.redd.it/6qi2513lfby21.jpg "Baby Bat")
